feeder/parsers/BG.py

```diff
@@ -0,0 +1,9 @@
+from ENTSOE import fetch_ENTSOE
+
+COUNTRY_CODE = 'BG'
+ENTSOE_DOMAIN = '10YCA-BULGARIA-R'
```

Comment:
Add in `feeder.py` instead of having a parser.
Also, make sure the add all the exchanges with neighboring countries